otx/core/data/manager/dataset_manager.py

```diff
@@ -31,12 +31,12 @@ def get_train_dataset(dataset: Dataset) -> DatasetSubset:
         raise ValueError("Can't find training data.")
 
     @staticmethod
-    def get_val_dataset(dataset: Dataset) -> DatasetSubset:
+    def get_val_dataset(dataset: Dataset) -> Union[DatasetSubset, None]:
         """Returns validation dataset."""
         for k, v in dataset.subsets().items():
-            if "val" in k or "default" in k:
+            if "val" in k:
                 return v
-        raise ValueError("Can't find validation data.")
+        return None
```

Comment:
return None -> It may be the cause of the current issue..?

Reply:
Thanks for investigation, I'll check it out

Reply:
we may delete return None since all procedures in Python return None by default

Reply:
Done, I deleted this line

Reply:
@kprokofi , no return is prohibited by mypy.. So, I left as is